### PR TITLE
docs: fixed inconsitency in @see usage in jsdoc

### PR DESCRIPTION
**Pull request by hankucz**

### CODE REVIEW

File: src/definitions/finance.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export type FinanceDefinitions = LocaleEntry<{
    * The pattern by (lowercase) issuer name used to generate credit card codes.
    * `L` will be replaced by the check bit.
    *
-   * @see Helpers.replaceCreditCardSymbols()
```
Comment (import-brain) [inline src/definitions/finance.ts:16]: Not too sure about this change, @Shinigami92, is there any special reason for it to be `Helpers` and not `faker.helpers`? I don't think so, but just making sure. Other than this, this PR LGTM

**Review by ejcheng — APPROVED** [review]

File: src/definitions/finance.ts (hunk 1 above)
Comment (ST-DDT) [reply src/definitions/finance.ts:16]: I like this change since it makes it consistent.

**Review by ST-DDT — COMMENTED** [review]
I'm not sure whether all methods need the empty `()`.

File: src/definitions/finance.ts (hunk 1 above)
Comment (import-brain) [reply src/definitions/finance.ts:16]: I like it too, just making sure there isn't any reason why it would be `Helpers`.

Comment (Shinigami92) [reply src/definitions/finance.ts:16]: No I don't know why `Helpers` was written here, change looks legit to me

**Comment by Shinigami92** [comment]
> I'm not sure whether all methods need the empty `()`.

How did we do it in the rest of the codebase and if it's mixed please let us decide one, document that and use it consistently

File: src/definitions/finance.ts (hunk 1 above)
Comment (import-brain) [reply src/definitions/finance.ts:16]: > No I don't know why `Helpers` was written here, change looks legit to me

Ah okay, thanks Shini

**Comment by ST-DDT** [comment]
I will write a test after this PR is merged.

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**